task_api/app/controllers/api/labels_controller.rb

```diff
@@ -34,11 +35,20 @@ def bulk_create
 
   private
 
+  def validate_task_exists
+    render json: { message: 'Specified Task Not Found', data: nil }, status: :not_found unless Task.exists?(id: params[:task_id])
+  end
+
   def set_label
+    label_name = params[:name]
+
+    label_name_exists = label_name and !label_name.gsub(' ', '').empty? # 空白だけの文字列も許可しない
+    return render json: { message: 'Label name is required', data: nil }, status: :bad_request unless label_name_exists
```

Comment:
何かの文字（スペース以外）が入っている場合のみ有効のチェックなら
↓でどうでしょう？

```suggestion
    return render json: { message: 'Label name is required', data: nil }, status: :bad_request if label_name.to_s.strip.empty?
```

Reply:
ありがとうございます！
かなりスマートになりますね
使用させて頂きます！